CITATION.cff

```diff
@@ -0,0 +1,11 @@
+cff-version: 1.2.0
+message: "If Iris played an important part in your research then please add us to your reference list by using the references below."
+title: "Iris"
+authors:
+    - name: "Iris contributors"
+abstract: "A powerful, format-agnostic, and community-driven Python package for analysing and visualising Earth science data"
+license: "LGPL-3.0"
```

Comment:
https://github.com/SciTools/iris/pull/5577

[`definitions.license-enum`](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#definitionslicense-enum)

I've run out of time to work out which one it should be! GeoVista has `BSD-3-Clause` so maybe check if we have identical wording.